src/dbus/dbus_object.cpp

```diff
@@ -33,6 +33,8 @@
 
 #include "dbus/dbus_object.hpp"
 
+using namespace std::placeholders;
```

Comment:
[Do not use using-directives](https://google.github.io/styleguide/cppguide.html#Namespaces)